pkg/vsphere/extraconfig/encode.go

```diff
@@ -19,22 +19,36 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"os"
 	"reflect"
 	"sort"
 	"strconv"
 	"strings"
 	"sync"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
+
+	"github.com/vmware/vic/pkg/log"
 )
 
 var (
-	// EncodeLogLevel value
-	EncodeLogLevel = log.InfoLevel
 	ErrKeyNotFound = errors.New("key not found")
 )
 
+var encodeLogger = &logrus.Logger{
```

Comment:
Both paths are very verbose and I only cared about one while debugging - didn't see a reason not to provide for this flexibility for now given it's trivial extra work at this point.